package.json

```diff
@@ -19,7 +19,7 @@
     "lint": "npm-run-all --parallel lint:*",
     "postinstall": "lerna bootstrap",
     "pretest": "npm run lint && node scripts/check-imports.js",
-    "test": "npm run test:unit && npm run test:closure && npm run test:dependency",
+    "test": "npm run test:unit && npm run test:closure && npm run test:dependency && npm run build",
```

Comment:
Should we append ` && npm run clean` to delete the generated `build/` directory?